filtering/checker.go

```diff
@@ -215,6 +222,10 @@ func (c *Checker) checkInt64Literal(e *expr.Expr) error {
 	return c.setType(e, TypeInt)
 }
 
+func (c *Checker) checkUint64Literal(e *expr.Expr) error {
```

Comment:
don't alias Uint to Uint64 - let's use consistent terminology across the board

Reply:
https://github.com/einride/aip-go/blob/00418014402ed1e97c6bda85bdbcc6804f4493b4/filtering/checker.go#L214-L216
I was referenced here, also alias Int to Int64, ,need modifications too?

Reply:
@odsod  PTAL ~